.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Test
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+    test:
+        name: Check code against linter/unit tests
+        runs-on: ubuntu-22.04
+        strategy:
+          matrix:
+            python-version: ['3.7', '3.8', '3.9', '3.10']
+        steps:
+        - uses: actions/checkout@master
+        - name: Set up Python ${{ matrix.python-version }}
+          uses: actions/setup-python@v4
+          with:
+            python-version: ${{ matrix.python-version }}
+        - name: Install dependencies
+          run: python -m pip install --upgrade "tox>=4.0.9,<5" build
+        - name: Build to generate the version file
+          run: python3 -m build
+        - name: Test with Tox
+          run: tox
```

Comment:
Could you please rename the file to pr_check.yml ?

Reply:
Is it possible to add support for python 3.11?

Reply:
It's worth to try, but what about to try it in a further PR to not overload the existing one?

Reply:
This is done